packages/paste-website/src/components/Heading.tsx

```diff
@@ -1,34 +1,48 @@
 import * as React from 'react';
-import {Heading, HeadingProps} from '@twilio-paste/heading';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Heading} from '@twilio-paste/heading';
+import type {HeadingProps} from '@twilio-paste/heading';
 import {styled, themeGet} from '@twilio-paste/styling-library';
+import {LinkIcon} from '@twilio-paste/icons/esm/LinkIcon';
 import {slugify} from '../utils/RouteUtils';
 
 const StyledAnchorHyperlink = styled.a`
   text-decoration: none;
   margin-left: ${themeGet('space.space30')};
-  color: ${themeGet('textColors.colorTextWeak')};
+  color: ${themeGet('textColors.colorTextIcon')};
 
   &:hover {
-    text-decoration: underline;
+    color: ${themeGet('textColors.colorTextLinkStronger')};
   }
 `;
 
+// const StyledAnchorHyperlink: React.FC = (children) => {
+//   return (
+//     <Text margin-left={themeGet('space.space30')} color={themeGet('textColors.colorTextWeak')} as="a">
+//       {children}
+//     </Text>
+//   );
+// };
+
 const AnchoredHeading: React.FC<HeadingProps> = (props) => {
```

Comment:
Should this component have VRT?

Reply:
Just added a story so that this can happen!